### PR TITLE
Add Web3auth

### PR DESCRIPTION
**Pull request by Rashmi-278**

### CODE REVIEW

**Review by kitakaze-kan — APPROVED** [review]
have put some comments, but almost LGFM! Thanks! 
Before merging, can you check only RPC stuff that I mentioned?

**Review by kitakaze-kan — APPROVED** [review]
LGFM! Thanks!

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -30,29 +32,53 @@ const global = css`
...
+  [infuraProvider({ apiKey: process.env.NEXT_PUBLIC_INFURA_KEY || '' }), publicProvider()],
+)
+
+// Instantiating Web3Auth
+const web3AuthInstance = new Web3Auth({
+  clientId: process.env.NEXT_PUBLIC_WEB3AUTH_CLIENT_ID || '',
+  chainConfig: {
+    chainNamespace: CHAIN_NAMESPACES.EIP155,
+    chainId: '0x' + chains[0].id.toString(16),
+    rpcTarget: chains[0].rpcUrls.default.http[0], // This is the public RPC we have added, please pass on your own endpoint while creating an app
```
[the remark this answers is not in the document]
Comment (Rashmi-278) [reply src/pages/_app.tsx]: Yes for sure, updated this

**Comment by kitakaze-kan** [comment]
@Rashmi-278 Thanks for checking and merging PR!

**Pull request merged**